### PR TITLE
Bug fix - IE side navigation

### PR DESCRIPTION
**Pull request by caleybrock**
# Description

Fixes side navigation on IE by:

- Removing all flex styles, since they are not totally compatible with IE
- Replacing _Flex_ by _Grid-Cell_ styles
- Making the Tab Navigation static (width of 1140, same as the default width)

## Type of change
- Bug fix (non-breaking change which fixes an issue)

## Screenshot

**Chrome:**

image

image

image

image

**IE:**

image

image

image

image

## Tests
image

# Checklist

Applicable for all code changes.

- [x] My code follows the style guidelines of this project
- [ ] ~~I have commented my code, particularly in hard-to-understand areas~~
- [ ] ~~I have made corresponding changes to the documentation~~
- [x] My changes generate no new compiler warnings
- [x] My changes generate no new accessibility errors and/or warnings
- [ ] ~~I have added tests that prove my fix is effective or that my feature works~~
- [x] New and existing unit tests pass locally with my changes
- [ ] ~~I have researched WCAG2.1 accessibility standards and met them in this PR (can be navigated using a keyboard)~~
- [x] My changes look good on IE 10+, Firefox, and Chrome

### CODE REVIEW

**Comment by fnormand01** [comment]
As discussed with @MichaelCherry offline, the text is going from left to right when you change from _Overview_, _Test Instruction_ and _Evaluation_ to _Tips on Taking the eMIB_.

**Overview:**

image

**Tips on Taking the eMIB:**

image

**Comment by MichaelCherry** [comment]
Just for the record: the "new vulnerability" detected by snyk was fixed in PR #107, which was merged to master. Snyk is detecting a difference between this package.json and the one on master, triggering the scan, and since CWE-471 is not present on master but is present here, it is getting flagged. CWE-471 will not be present in master when this branch is merged.

File: frontend/src/css/cat-theme.css [hunk 1]
```diff
@@ -145,3 +145,20 @@ Progess Steps
 .nav-pills .show > .nav-link {
   background-color: #00565e;
 }
+
+/* Side Navigation */
+
+.side-nav-grid {
+  display: -ms-grid;
```
Comment (MichaelCherry) [inline frontend/src/css/cat-theme.css]: Just curious: why are we putting these in the css and not in a style const?

Comment (fnormand01) [reply frontend/src/css/cat-theme.css]: As discussed offline, I did not find a way to put this type of display in a const. I tried putting like _display: "-ms-grid"_ but it did not work. It seems like we need to put this specific type of display in a _.css_ file to be recognized.

@caleybrock, do you have an idea on how to put that in the component itself?

**Comment by MichaelCherry** [comment]
Only concerns are the one mentioned above (styles in the css rather than style; but I understand from our discussion offline why they are there)
And this: content portion does not resize well. Maybe using max width rather than width?
image

**Comment by caleybrock** [comment]
I'll do a review of this this morning.

File: frontend/src/css/cat-theme.css (hunk 1 above)
Comment (caleybrock) [reply frontend/src/css/cat-theme.css]: Can you make another CSS file called `side-nav.css` with this new CSS and then import that file into SideNavigation.jsx?

Comment (fnormand01) [reply frontend/src/css/cat-theme.css]: Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/103/commits/eea0ce44bd90b0a1fa3b852972de8ed35f120dc1

**Comment by caleybrock** [comment]
Changes look good to me.

**Pull request merged**